apps/cookbook/src/app/showcase/list-showcase/list-showcase.component.html

```diff
@@ -135,3 +135,7 @@ <h2>List with sections and standalone items:</h2>
     ></cookbook-iphone>
   </div>
 </div>
+
```

Comment:
I would suggest that you display the default value first

Reply:
Yes sir